.circleci/pr-skip-ci.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+# This script makes CircleCI exit a non pull-request job.
+
+echo "CIRCLE_PULL_REQUEST: ${CIRCLE_PULL_REQUEST}"
+regexp="[[:digit:]]\+$"
```

Comment:
Are you scanning the PR description for a Jira Ticket number?

I thought CircleCI already had a away to disable it for all but pull requests, but I remember we decided not to do that anymore for some reason.

Reply:
No, not scanning PR description for ticket number.
The CircleCI setting to pull on PR only is still enabled, no change there.
<img width="1027" alt="Screen Shot 2020-06-24 at 11 43 17 AM" src="https://user-images.githubusercontent.com/35533885/85587779-136f1380-b610-11ea-9d50-5c3f8dd645ea.png">